libs/ui/src/lib/components/forms/GenomicForm.tsx

```diff
@@ -88,8 +89,34 @@ export const GenomicForm = (props: IFormProps) => {
 
   const dispatch = useDispatch()
 
+const hugo_gene_component = () => {
+  const [hugoSymbols, setHugoSymbols] = useState([]);
+    useEffect(() => {
+      async function fetchTopSymbols() {
+        try {
+          const response = await axios.get('http://localhost:3333/api/genes') 
```

Comment:
You could consider using useAxios hook under web/hooks

Reply:
would that look like this? 

`axios.defaults.baseURL = 'http://localhost:3333/api';

const useAxios = () => {
    const [hugoSymbols, setHugoSymbols] = useState([]);
    const [error, setError] = useState('');

    const fetchHugoSymbols = () => {
        axios
            .get('/genes')
            .then(function (response) {
            const symbols = response.data;
            setHugoSymbols(symbols);
            hugoSymbols.push(symbols);
          })
            .catch((err) => {
                setError(err);
            })
    };

    useEffect(() => {
        fetchHugoSymbols();
    }, []);

    return hugoSymbols;
};`

Reply:
@Singhj2 this commit includes the changes I think we can use https://github.com/Singhj2/ctims/commit/3eb8781175f12979e09d9f5381826c3bdc9dfea9

Sorry I accidentally pushed the changes to your branch, but I reverted it. Therefore the two additional commits.